plugin/storage/es/Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM python:3-alpine3.11
 
-# Temporary fix for https://github.com/jaegertracing/jaeger/issues/1494
-RUN pip install urllib3==1.24.3
-# Temporary fix for https://github.com/jaegertracing/jaeger/issues/3179
-RUN pip install elasticsearch==7.13.0
+COPY requirements-build.txt requirements-build.txt
+COPY requirements.txt requirements.txt
```

Comment:
if we're always copying both files in prod output there is no reason to separate build from main requirements, since they are both required for prod.

Reply:
Correct, but downstream (like our Dockerfiles) might run on hermetic environments so that we need to ensure that all build requirements are installed before we install the runtime dependencies. For instance, the "cython" requirement may be resolved to be installed after "elasticsearch-curator", as the latter doesn't list the former in its list of (runtime) dependencies.

We use "Cachito" internally, and this is a link to the explanation on their side: https://github.com/release-engineering/cachito/blob/master/docs/pip.md#build-dependencies